wolfssl/wolfio.h

```diff
@@ -920,7 +926,12 @@ WOLFSSL_API void wolfSSL_SetIOWriteFlags(WOLFSSL* ssl, int flags);
         #if defined(__MINGW64__) && !defined(UNICODE)
             #define XINET_PTON(a,b,c)   InetPton((a),(b),(c))
         #else
-            #define XINET_PTON(a,b,c)   InetPton((a),(PCWSTR)(b),(c))
+            #if (defined(_MSC_VER) && (_MSC_VER >= 1600)) || \
+                 defined(_WIN32_WCE) || defined(WINCE) || defined(PocketPC)
```

Comment:
Just use `_WIN32_WCE`. You don't need WINCE or PocketPC

Reply:
Ok, I'll used only `_WIN32_WCE`. It may need to be manually defined for some projects.